apps/dh/e2e-dh/src/integrations/language-selection.spec.ts

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
suggestion: there are two folders with identical names and tests -> "integration" and "integrations". Delete one of the folders.

Reply:
Should be removed now